receiver/mongodbreceiver/metrics.go

```diff
@@ -351,6 +352,7 @@ func (s *mongodbScraper) recordCursorCount(now pcommon.Timestamp, doc bson.M, er
 	cursorCountValue, err := parseInt(cursorCount)
 	if err != nil {
 		errs.AddPartial(1, err)
+		return
```

Comment:
Are these returns/continue related?

Reply:
Without that commit, then in the case that they would get triggered they would cause 2 partial errors or emit metrics of the wrong type. Related to partial errors, but not metric version check.

Reply:
Removed to add at later date.